### DISCUSSION TITLE
Candeo Zigbee Dimmer unrecognised

### DISCUSSION THREAD

**Discussion by stevepbuk** [Koenkk/zigbee2mqtt#16789] [Devices]
Hi,

I have just purchased a Candeo Zibee Dimmer and tried to connect it to Home Assistant through Zigbee2MQTT. This has been officially supported since November and it looks like pretty much everyone has managed to get it to work straight away. I am running into difficulty however.

The device is paired with Zigbee2MQTT, however it is stating that its ‘Unsupported’ and therefore I canot get it added to Home Assitant. Nearly every review I have read people are linking this without any problems. I have also tried resetting the dimmer to defaults and trying again, but I get the same outcome. Has anyone had the same device and had this issue and found a resolution.

The Light switch firmware date is 2022/05/31 which is reported through Zigbee2MQTT. The Zigbee device I am using is a SONOFF Universal Zigbee 3.0 USB Dongle which is connected to a Raspberry Pi 4. I am running the latest version of Home Assistant and 1.30.1-1 of Zigbee2MQTT.

The log from Zigbee2MQTT can be found below:

Info 2023-02-22 21:44:36Device '0x94deb8fffe5e953a' joined
Info 2023-02-22 21:44:36MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x94deb8fffe5e953a","ieee_address":"0x94deb8fffe5e953a"},"type":"device_joined"}'
Info 2023-02-22 21:44:36Starting interview of '0x94deb8fffe5e953a'
Info 2023-02-22 21:44:36MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x94deb8fffe5e953a","ieee_address":"0x94deb8fffe5e953a","status":"started"},"type":"device_interview"}'
Debug 2023-02-22 21:44:47Device '0x94deb8fffe5e953a' announced itself
Info 2023-02-22 21:44:47MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x94deb8fffe5e953a","ieee_address":"0x94deb8fffe5e953a"},"type":"device_announce"}'
Debug 2023-02-22 21:44:50Received Zigbee message from '0x94deb8fffe5e953a', type 'readResponse', cluster 'genBasic', data '{"powerSource":178}' from endpoint 1 with groupID 0
Debug 2023-02-22 21:44:50Skipping message, definition is undefined and still interviewing
Debug 2023-02-22 21:44:50Received Zigbee message from '0x94deb8fffe5e953a', type 'readResponse', cluster 'genBasic', data '{"appVersion":0}' from endpoint 1 with groupID 0
Debug 2023-02-22 21:44:50Skipping message, definition is undefined and still interviewing
Debug 2023-02-22 21:44:50Received Zigbee message from '0x94deb8fffe5e953a', type 'readResponse', cluster 'genBasic', data '{"stackVersion":0}' from endpoint 1 with groupID 0
Debug 2023-02-22 21:44:50Skipping message, definition is undefined and still interviewing
Debug 2023-02-22 21:44:50Received Zigbee message from '0x94deb8fffe5e953a', type 'readResponse', cluster 'genBasic', data '{"hwVersion":0}' from endpoint 1 with groupID 0
Debug 2023-02-22 21:44:50Skipping message, definition is undefined and still interviewing
Info 2023-02-22 21:44:50Successfully interviewed '0x94deb8fffe5e953a', device has successfully been paired
Warning 2023-02-22 21:44:50Device '0x94deb8fffe5e953a' with Zigbee model 'Candeo Zigbee Dimmer' and manufacturer name 'Candeo' is NOT supported, please follow https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html

Thanks

Steve

**Comment by Koenkk** [comment]
This device is only supported in the dev version and will be supported in the next release on 1 February.

**Reply by SteveDockar** [reply]
This wasn't working for me in 1.30.2 but thanks to the efforts, and the external converter fix described in [#16613](https://github.com/Koenkk/zigbee2mqtt/issues/16613) which seems to have fixed it. 

Will the external converter need removing to avoid issues when this rolls in to the main release branch @Koenkk or will the OOTB support still work with this in configuration.yml?

**Reply by Koenkk** [reply]
You can keep using the external converter but I recommend to remove it after updating to the next release.

**Reply by SteveDockar** [reply]
Thanks, @Koenkk that's perfect. 

I was worried that having both would cause an issue _during_ the upgrade, so that I'd need to check the release notes to see when the fix arrived and coordinating removing the external converter just before I applied the update.

**Reply by Koenkk** [reply]
Nothing will break when you keep using it, so just remove after or right before you update.
